api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -42,10 +42,6 @@ public interface ProfileMapper {
   @Mapping(source = "dbUser.duccAgreement.userInitials", target = "duccSignedInitials")
   @Mapping(source = "dbUser.duccAgreement.completionTime", target = "duccCompletionTimeEpochMillis")
   @Mapping(source = "dbUser.demographicSurveyV2", target = "demographicSurveyV2")
-  @Mapping(
```

Comment:
Why remove this?  We're eventually going to need it on the Profile page at least

Reply:
I believe that this was an accident. Thanks for catching it.